tools/python/boutiques/schema/descriptor.schema.json

```diff
@@ -516,6 +516,15 @@
                 ]
             }
         },
+        "online-platform-urls": {
+            "id": "http://github.com/boutiques/boutiques-schema/online-platforms",
+            "type": "array",
+            "description": "Online platform URLs from which the tool can be executed.",
+            "items": {
+                "type": "string",
+                "pattern": "^https?://"
```

Comment:
Question. Does the schema validator applies the regex using multiline mode? If so, it would allow a descriptor to contain a string such as `"bouahaha\nhttps://example.com/"` and still calls it 'valid', because the caret would match the zero-width character after the newline.

Reply:
It looks fine:
`
[ ERROR ] 'bouahaha\nhttps://example.com/' does not match '^https?://'

Failed validating 'pattern' in schema['properties']['online-platform-urls']['items']:
    {'pattern': '^https?://', 'type': 'string'}

On instance['online-platform-urls'][2]:
    'bouahaha\nhttps://example.com/'
`